msf_updates/include/msf_updates/pressure_sensor_handler/implementation/pressure_sensorhandler.hpp

```diff
@@ -30,8 +30,8 @@ PressureSensorHandler::PressureSensorHandler(
   ros::NodeHandle pnh("~/pressure_sensor");
   ros::NodeHandle nh("msf_updates");
   subPressure_ =
-      nh.subscribe<asctec_hl_comm::mav_imu>
-      ("pressure_input", 20, &PressureSensorHandler::MeasurementCallback, this);
+      nh.subscribe<geometry_msgs::PointStamped>
+      ("pressure_height", 20, &PressureSensorHandler::MeasurementCallback, this);
```

Comment:
good point, I think [here](https://github.com/ethz-asl/ethzasl_msf/blob/master/msf_updates/launch/pose_pressure_sensor.launch#L6) is another occurrence, but I would prefer if we could quickly try if everything works on an actual platform, before merging this.